packages/flutter/lib/src/material/text_field.dart

```diff
@@ -459,6 +461,11 @@ class TextField extends StatefulWidget {
   /// {@end-tool}
   final InputCounterWidgetBuilder buildCounter;
 
+  /// The [ScrollPhysics] to use when vertically scrolling the TextField.
+  ///
+  /// See [Scrollable.physics].
```

Comment:
Need to mention the defualt

Reply:
Looking into this, I just noticed that Scrollable has the ability to choose its physics based on the current platform.  Rather than hardcoding Material to Clamping and Cupertino to Bouncing, I just let `null` get passed to Scrollable so that it can handle the default.  The result is that iOS defaults to Bouncing and Android to Clamping regardless of Material or Cupertino.  That seems like the best behavior to me, what do you think?

Reply:
That's much better!